ui-tests/package.json

```diff
@@ -10,8 +10,8 @@
     "test:update": "jlpm playwright test --update-snapshots"
   },
   "devDependencies": {
-    "@jupyterlab/galata": "^4.3.0",
-    "@playwright/test": "^1.34.2",
+    "@jupyterlab/galata": "^5.0.5",
```

Comment:
if upgrading galata breaks things, maybe we can keep the old version? I think it's better to upgrade but if it breaks stuff, we might consider keeping an older version for now

Reply:
Galata 5 is the version from Jupyterlab 4, I think it's better to upgrade it, there could be some side effect to keep the old version.